tests/test_bibliotheca.py

```diff
@@ -473,7 +473,9 @@ def test_fulfill(self):
             200, headers={"Content-Type": "presumably/an-acsm"},
             content="this is an ACSM"
         )
-        fulfillment = self.api.fulfill(patron, 'password', pool, 'ePub')
+        fulfillment = self.api.fulfill(
+            patron, 'password', pool, internal_format='ePub'
```

Comment:
Here I'm changing the `fulfill` call to make it more closely resemble the call that's actually going to come in from `CirculationAPI`.